src/Hooks/useEdit.tsx

```diff
@@ -0,0 +1,98 @@
+import { useContext, useRef } from 'react';
+import { Todo } from '../Type/Todo';
+import { Edit } from '../Enum/Edit';
+import { TodoListContext } from '../Context/TodoListContext';
+
+export const useEdit = (todos: Todo[]) => {
+  const editedTodoList = useRef<Todo[]>(todos);
+  const { todoList, setTodoList } = useContext(TodoListContext);
+
+  const getStatus = (list: Todo[]) => {
+    if (list.every(todoItem => todoItem.completed)) {
+      return false;
+    } else if (list.every(todoItem => !todoItem.completed)) {
+      return true;
+    } else {
+      return true;
+    }
```

Comment:
```suggestion
    if (list.every(todoItem => todoItem.completed)) {
      return false;
    } 
    
    return true;
```
It should work the same